doxygen/contributor_help_pages/require_meta.md

```diff
@@ -0,0 +1,191 @@
+## Using requires<> for general overloads. {#require_meta_doc}
+
+The [`requires`](@ref require_meta) type traits are aliases for `std::enable_if_t` that have premade
+conditions for turning on and off function definitions during compilation.
+These are useful for having generalized templates while still overloading a function or class.
+You can think of these as "legacy concepts" aka I think the closest thing you can get to C++20 concepts without having the `requires` keyword.
+
+`requires` are [`std::enable_if_t`](https://en.cppreference.com/w/cpp/types/enable_if) aliases such as the below definition of @ref stan::require_t.
+
+```cpp
+template <typename T>
+using require_t = std::enable_if_t<T::value>;
+```
+
+Note that unlike `std::enable_if_t`, the alias @ref stan::require_t 's template type `T`
```

Comment:
I couldn't parse this sentence.   Could you maybe try rewriting it?  Or could you just give an example of what must have a valid member value?  Is it the argument to `enable_if_t`, perhaps?

Reply:
I rewrote this to try to make it more sensible